src/routes/_surfaces/workspace/explore/time-series-charts/MetricsTimeSeriesCharts.svelte

```diff
@@ -80,12 +79,12 @@
   let timeSeriesQueryKey = getMetricViewTimeSeriesQueryKey(metricsDefId);
   let timeSeriesQueryFn = () =>
     getMetricViewTimeSeries(metricsDefId, {
-      measures: $metricsExplorer.measureIds,
-      filter: $metricsExplorer.filters,
+      measures: $queryResult.data.measures.map((measure) => measure.id),
```

Comment:
Shouldn't these measures come from the client-side store (`metricsExplorer.selectedMeasures`) not from the server?

Reply:
The meta returns valid measures. Client side can have invalid ones as well. Just wanted to move to a pattern here that always calls the backend.

Reply:
Actually you are right.. We need to filter measures from server based on selectedMeasures.

Reply:
Done